e2e/path/zonefile

```diff
@@ -90,28 +90,28 @@ _redirect.3.predefined-multinumbered.host.path.example.com. IN TXT "v=txtv0;type
 _redirect.2.predefined-simpletospecific.host.path.example.com. IN TXT "v=txtv0;type=host;to=https://second-record.host.path.example.com/{2};re=^/([^/]*)/?$"
 
-predefined.host.path.example.com.             IN A 172.20.10.2
-_redirect.predefined.host.path.example.com.   IN TXT "v=txtv0;type=path;root=https://index.host.path.example.com;re=record"
-_redirect.1.predefined.host.path.example.com. IN TXT "v=txtv0;type=host;to=https://predefined-regex.host.path.example.com/1;re=^/api-review$"
-_redirect.2.predefined.host.path.example.com. IN TXT "v=txtv0;type=host;to=https://predefined-regex.host.path.example.com/2;re=^/bot-commands$"
-_redirect.3.predefined.host.path.example.com. IN TXT "v=txtv0;type=host;to=https://predefined-regex.host.path.example.com/3;re=^/github-labels$"
-_redirect.4.predefined.host.path.example.com. IN TXT "v=txtv0;type=host;to=https://predefined-regex.host.path.example.com/4;re=^/good-first-issue$"
-_redirect.5.predefined.host.path.example.com. IN TXT "v=txtv0;type=host;to=https://predefined-regex.host.path.example.com/5;re=^/help-wanted$"
-_redirect.6.predefined.host.path.example.com. IN TXT "v=txtv0;type=host;to=https://predefined-regex.host.path.example.com/6;re=^/needs-ok-to-test$"
-_redirect.7.predefined.host.path.example.com. IN TXT "v=txtv0;type=host;to=https://predefined-regex.host.path.example.com/7;re=^/oncall$"
-_redirect.8.predefined.host.path.example.com. IN TXT "v=txtv0;type=host;to=https://predefined-regex.host.path.example.com/8;re=^/oncall-hotlist$"
+predefined.host.path.example.com.                              IN A 172.20.10.2
+_redirect.predefined.host.path.example.com.                    IN TXT "v=txtv0;type=path;root=https://index.host.path.example.com"
+_redirect.api-review.predefined.host.path.example.com.         IN TXT "v=txtv0;type=host;to=https://predefined-regex.host.path.example.com/1"
+_redirect.bot-commands.predefined.host.path.example.com.       IN TXT "v=txtv0;type=host;to=https://predefined-regex.host.path.example.com/2"
+_redirect.github-labels.predefined.host.path.example.com.      IN TXT "v=txtv0;type=host;to=https://predefined-regex.host.path.example.com/3"
+_redirect.good-first-issue.predefined.host.path.example.com.   IN TXT "v=txtv0;type=host;to=https://predefined-regex.host.path.example.com/4"
+_redirect.help-wanted.predefined.host.path.example.com.        IN TXT "v=txtv0;type=host;to=https://predefined-regex.host.path.example.com/5"
+_redirect.needs-ok-to-test.predefined.host.path.example.com.   IN TXT "v=txtv0;type=host;to=https://predefined-regex.host.path.example.com/6"
+_redirect.oncall.predefined.host.path.example.com.             IN TXT "v=txtv0;type=host;to=https://predefined-regex.host.path.example.com/7"
+_redirect.oncall-hotlist.predefined.host.path.example.com.     IN TXT "v=txtv0;type=host;to=https://predefined-regex.host.path.example.com/8"
```

Comment:
It's not predefined anymore. So we should probably change the sub zone to something closer to reality.